source/_posts/2025-03-05-release-20253.markdown

```diff
@@ -0,0 +1,563 @@
+at, and don't forget to join our amazing [forums](https://community.home-assistant.io/).
+
+Found a bug or issue? Please report it in our [issue tracker](https://github.com/home-assistant/core/issues),
+to get it fixed! Or, check [our help page](/help) for guidance for more
+places you can go.
+
+Are you more into email? [Sign-up for our Building the Open Home Newsletter](/newsletter)
+to get the latest news about features, things happening in our community and
+other news about building an Open Home; straight into your inbox.
+
+## Backward-incompatible changes
+
+We do our best to avoid making changes to existing functionality that might
+unexpectedly impact your Home Assistant installation. Unfortunately, sometimes,
+it is inevitable.
+
+We always make sure to document these changes to make the transition as easy as
+possible for you. This release has the following backward-incompatible changes:
+
+{% details "GPSD" %}
+
+All state attributes of the main sensor have been replaced with dedicated
+sensor Home Assistant 2024.9. The deprecated attributes have now been removed.
+
+([@jrieger] - [#137600]) ([documentation](/integrations/gpsd))
+
+[@jrieger]: https://github.com/jrieger
+[#137600]: https://github.com/home-assistant/core/pull/137600
+
+{% enddetails %}
+
+{% details "Home Connect" %}
+
+Some programs at program switches (which are being deprecated) and program
+select entities may not be exposed anymore by the integration as they are
+missing at `aiohomeconnect` program key enumeration ([file](https://github.com/MartinHjelmare/aiohomeconnect/blob/ccfbf802884aa781eef481da92e6bccd2e6de733/src/aiohomeconnect/model/program.py#L213C1-L507C6)).
+You can get the diagnostics from configuration entry or the device and open an
+issue or submit a PR with the generated diagnostics file on the
+[`aiohomeconnect` repository](https://github.com/MartinHjelmare/aiohomeconnect)
+to suggest the missing program(s).
+
+([@Diegorro98] - [#136116]) ([documentation](/integrations/home_connect))
+
+[@Diegorro98]: https://github.com/Diegorro98
+[#136116]: https://github.com/home-assistant/core/pull/136116
+
+---
+
+Using program and option keys that aren't at the API documentation, will no
+longer work. To be able to use them, open an issue or submit a pull request
+with the required keys at [MartinHjelmare/aiohomeconnect](https://github.com/MartinHjelmare/aiohomeconnect).
+
+([@Diegorro98] - [#137027]) ([documentation](/integrations/home_connect))
+
+[@Diegorro98]: https://github.com/Diegorro98
+[#137027]: https://github.com/home-assistant/core/pull/137027
+
+---
+
+Multiple Home Connect config entries will not be allowed to be configured. For
+regular users where all appliances in the home are connected to the same Home
+Connect account there should not be any reason to have more than one config
+entry since all appliances in the home will be connected to the same account
+and config entry.
```

Comment:
```suggestion
Multiple Home Connect config entries will not be allowed to be configured. For
regular users where all appliances in the home are connected to the same Home
Connect account, there should not be any reason to have more than one config
entry since all appliances in the home will be connected to the same account
and config entry.
```